Update for 1.5 profiles (#6547)
istio/istio.io: content/en/docs/setup/additional-setup/config-profiles/index.md

1. **minimal**: the minimal set of components necessary to use Istio's [traffic management](/docs/tasks/traffic-management/) features.

1. **remote**: used for configuring remote clusters of a
    [multicluster mesh](/docs/ops/deployment/deployment-models/#multiple-clusters) with a
    [shared control plane](/docs/setup/install/multicluster/shared-vpn/) configuration.

The components marked as **X** are installed within each profile:

> |     | default | demo | minimal | remote |

Reviewer: Do we need to create content to mention `empty` and `separate`?

Reviewer: We should probably get this one in sooner than later. Any thoughts on adding the additional columns?

1. **minimal**: the minimal set of components necessary to use Istio's [traffic management](/docs/tasks/traffic-management/) features.

1. **remote**: used for configuring remote clusters of a
    [multicluster mesh](/docs/ops/deployment/deployment-models/#multiple-clusters) with a
    [shared control plane](/docs/setup/install/multicluster/shared-vpn/) configuration.

> 1. **preview**: this profile enables experimental features that may not meet the same level of stability, performance, and security as the default profiles.

Reviewer 2: ```suggestion
1. **preview**: enables experimental features that lack the stability, performance, and security of features in the default profiles.
```

Author: nit: To me this makes it sound like they are all unstable, bad performance, and not secure. Do you agree? I think change `and` to `or` or `and/or` may make more sense. Let me know what you think

> 1. **empty**: deploys nothing. This can be useful as a base profile for custom configuration.

Reviewer 2: The phrase `deploys nothing.` can be confusing for readers. They might ask: Will the command do *nothing*? Which is not what happens. This profile just means that no configuration profile is applied, correct? The text should let readers know that's the case.

Author: It literally does nothing. Its not the same as no profile, which would mean the default one. It was added so you can do things like `--set profile=empty --set something.enabled=true` instead of needing to go and disable all the default stuff.

I didn't want to document it since its confusing but I got recommended to add all profiles here

Reviewer 3: I think instead of `deploys nothing`,  the following would be better.  

`The empty profile has everything disabled.`

> 1. **separate**: this enables the legacy microservices deployment model. This is not recommended, and will be removed in future releases.

Reviewer 2: ```suggestion
1. **separate**: deploys Istio following the legacy micro-services model. This profile is not recommended because it won't be supported in future releases.
```
